.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: build
+
+on: [push, pull_request]
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
```

Comment:
I think Gradle's recommendation is to use daemon in CI too nowadays so maybe don't need this

Reply:
Done.